### ISSUE TITLE
Cannot compile, probable incompatibility with gcc 11

### ISSUE THREAD

**Issue by alerque** [praat/praat#1749]
I just tried bumping the [Arch Linux packaging](https://aur.archlinux.org/packages/praat/) to 6.1.44 and the build almost immediately fails. This is a pretty generic build [using](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=praat#n33) the linux/pulse profile and a simple `make`.

The most likely culprit I can think of is that Arch's `gcc` just recently got updated from 10.2 to 11.1.

```
make -C external/clapack
make[1]: Entering directory '/home/caleb/projects/aur/praat/src/praat-6.1.44/external/clapack'
g++ -std=c++17 -DUNIX -Dlinux -DALSA -DHAVE_PULSEAUDIO -D_FILE_OFFSET_BITS=64 `"pkg-config" --cflags gtk+-3.0` -Wreturn-type -Wunused -Wunused-parameter -Wuninitialized -O3 -g1 -pthread -Wshadow -I ../../melder  -c -o blas.o blas.cpp
In file included from ../../melder/melder.h:52,
                 from cblas.h:21,
                 from blas.cpp:1:
../../melder/melder_real.h:61:27: error: ‘numeric_limits’ is not a member of ‘std’
   61 |         double min = std::numeric_limits<double>::max();
      |                           ^~~~~~~~~~~~~~
../../melder/melder_real.h:61:42: error: expected primary-expression before ‘double’
   61 |         double min = std::numeric_limits<double>::max();
      |                                          ^~~~~~
../../melder/melder_real.h:62:27: error: ‘numeric_limits’ is not a member of ‘std’
   62 |         double max = std::numeric_limits<double>::lowest();
      |                           ^~~~~~~~~~~~~~
../../melder/melder_real.h:62:42: error: expected primary-expression before ‘double’
   62 |         double max = std::numeric_limits<double>::lowest();
      |                                          ^~~~~~
make[1]: *** [<builtin>: blas.o] Error 1
make[1]: Leaving directory '/home/caleb/projects/aur/praat/src/praat-6.1.44/external/clapack'
make: *** [Makefile:34: all-external] Error 2
```

**Comment by rlaboiss** [comment]
Please, see PR #1622.

**Comment by alerque** [comment]
Okay great. Is there some reason that has not been merged yet? IF the fix has been known for a few months and it's a one line include issue, surely...

I'm cat patch this to get Arch packages building again, but it would make more sense for this to get fixed upstream, wouldn't it?

**Comment by rlaboiss** [comment]
I am also applying the patch for the Debian packaging. I think that the Praat authors will eventually catch up. Just be patient.

**Comment by rlaboiss** [comment]
I think that this issue can be closed, because commit b4aea3019bdc102eb0c1fba1d48734f9c4f6dedb fixes it.

**Comment by alerque** [comment]
Ya I bumped into not being able to build Arch packages yesterday because the patch wasn't applying cleanly. I just looked at it and it's because the same patch is applied.

For maintainers, next time it would be easier if you merged PRs instead of applying patches out of band. Git has mechanisms for this and our builds would have detected the PR being applied (even if rebased or cherry-picked) but it didn't understand the code being changed in an unrelated commit history.
